ledger/participant-integration-api/src/main/scala/platform/LedgerApiServer.scala

```diff
@@ -147,6 +152,12 @@ class LedgerApiServer(
         cacheExpiryAfterWrite = apiServerConfig.identityProviderManagement.cacheExpiryAfterWrite,
         maxIdentityProviders = IdentityProviderManagementConfig.MaxIdentityProviders,
       )(servicesExecutionContext, loggingContext)
+    val identityProviderConfigLoader = new IdentityProviderConfigLoader {
+      override def getIdentityProviderConfig(issuer: LedgerId)(implicit
+          loggingContext: LoggingContext
+      ): Future[domain.IdentityProviderConfig] =
+        identityProviderStore.getActiveIdentityProviderByIssuer(issuer)
```

Comment:
which execution context should this run on?

Reply:
servicesExecutionContext ?

Reply:
I would think so too